README.md

```diff
@@ -147,7 +147,11 @@ make e2e
 
 ## Configuration
 
-See [examples/config.yaml](./examples/config.yaml).
+There is a default configuration in [examples/config.yaml](./examples/config.yaml).
```

Comment:
"default" means even if you don't change anything, it should work.

I think "This is an example of DevStream config" is more suitable here.